### ISSUE TITLE
fix link and image markup in "Mosaic Listing": 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#338]
In order to meet accessibility criteria, we need to address the appearance of redundant links and undesireable ```title``` attributes in the "Mosaic Listing". 

At present, each item represented in the "Mosaic Listing" gets a ```<div class="ExhibitionEntry">```, structured as follows:

```html
<div class="ExhibitionEntry">
  <div class="ExhibitionEntryImage">
    <a href="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/conical-sundial-oscan" title="Conical Sundial with Oscan Dedicatory Inscription">
      <img src="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/conical-sundial-oscan/@@images/5bdaa622-3ce3-4441-8189-0ed907fbcc32.jpeg" alt="Conical Sundial with Oscan Dedicatory Inscription" title="Conical Sundial with Oscan Dedicatory Inscription" width="768" height="629">
    </a>
  </div>
  <div class="ExhibitionEntrySummary">
    <a href="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/conical-sundial-oscan">
      <h3>Conical Sundial with Oscan Dedicatory Inscription</h3>
    </a>
  </div>
</div>
```

The encoding for each of these ```<div>```s needs to be changed in order to:

1. Eliminate the use of the ```title``` attribute on links (```<a>```) and images (```<img>```).
2. Create one and only one link (```<a>```) to a given exhibition object (right now there are two: one around the image and the other around the object title, which is in ```<h3>```)
3. Create an empty ```alt``` element on each image so that they are not read out by a screen reader (because the purpose of this page is to get the user to the specific information page about each object).

The resulting encoding should look like this:

```html
<div class="ExhibitionEntry">
  <a href="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/conical-sundial-oscan">
    <div class="ExhibitionEntryImage">
      <img alt=""
           src="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/conical-sundial-oscan/@@images/5bdaa622-3ce3-4441-8189-0ed907fbcc32.jpeg" 
           width="768" height="629">
    </div>
    <div class="ExhibitionEntrySummary">
      <h3>Conical Sundial with Oscan Dedicatory Inscription</h3>
    </div>
  </a>
</div>
```

It may be the case that #293 will take care of the ```title``` attributes (or not), so it should be considered a blocker for this issue.

Note that visual look and feel should remain the same, so some CSS adjustment may be required if the markup change alters the layout.

**Comment by cguardia** [comment]
This is on staging now.

**Comment by paregorios** [comment]
This looks great! Please merge to master and deploy to production.
